modules/setting/annex.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package setting
+
+import (
+	"code.gitea.io/gitea/modules/log"
+)
+
+// LFS represents the configuration for Git LFS
```

Comment:
Should update `LFS` -> `Annex`:
```suggestion
// Annex represents the configuration for git-annex
```